loopd/swapclient_server.go

```diff
@@ -645,6 +645,34 @@ func rpcToRule(rule *looprpc.LiquidityRule) (*liquidity.ThresholdRule, error) {
 
 }
 
+// SuggestSwaps provides a list of suggested swaps based on lnd's current
+// channel balances and rules set by the liquidity manager.
+func (s *swapClientServer) SuggestSwaps(ctx context.Context,
+	_ *looprpc.SuggestSwapsRequest) (*looprpc.SuggestSwapsResponse, error) {
+
+	swaps, err := s.liquidityMgr.SuggestSwaps(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	var loopOut []*looprpc.LoopOutRequest
+
+	for _, swap := range swaps {
+		loopOut = append(loopOut,
+			&looprpc.LoopOutRequest{
```

Comment:
nit: could move `&looprpc.LoopOutRequest` to previous line, would get rid of one level of indentation.